web/WEB-INF/views/index.jsp

```diff
@@ -1,16 +1,51 @@
-<%--
-  Created by IntelliJ IDEA.
-  User: vsaban
-  Date: 3/27/2017
-  Time: 11:43 AM
-  To change this template use File | Settings | File Templates.
---%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
 <html>
 <head>
     <title>Home Page</title>
 </head>
 <body>
-    <h3>Hello, World!</h3>
+    <p>Hello, user!</p>
+
+    <table>
+
+        <tr>
+            <th>Wherefroms</th>
+            <th>Domains</th>
+        </tr>
+        <tr>
+            <td>
```

Comment:
Remove `<br />` from `<td></td>`

it's ok if you will add in the future somethink like:
```html
<c:forEach items="${domains}" var="item">
     <tr> 
         <td>${item.name}</td>
         <td>${item.value}</td>
         <td>....</td>
     </tr>
</c:forEach>
```